app/components/primer/alpha/banner.rb

```diff
@@ -3,6 +3,13 @@
 module Primer
   module Alpha
     # Use `Banner` to highlight important information.
+    #
+    # @accessibility
+    #   Depending on the scenario, some Banners may need to receive focus when they appear. This helps to maximize discoverability of the message, especially in critical scenarios. Defer to the accessibility team to determine if your scenario requires focusing the banner.
+    #
```

Comment:
When we have the guidance for live region, I could see each under a `Focus` and a `Live region announcement` sub section.

Reply:
Yes! I think so too! I am not sure if sections are supported in the markdown comments (I am still trying to spin this up locally) but I can keep poking.

Reply:
OOH to be clear, what you have looks good to me and I can update the format when I add the live region section as well!